trie/zk_merkle_trie_test.go

```diff
@@ -0,0 +1,85 @@
+package trie
```

Comment:
I wonder why you named this test file not `zk_merkle_state_trie_test.go`.

Reply:
I named it that way because I'm going to handle three tests (zk_merkle_stack_trie.go, zk_merkle_state_trie.go, zk_merkle_trie.go) in one file.